frontend/src/components/MapView/Legends/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { LegendDefinitionItem } from 'config/types';
+
+// Invert the colors of the legend, first color becomes last and vice versa
+export const invertLegendColors = (
+  legendItems: LegendDefinitionItem[],
+): LegendDefinitionItem[] => {
+  // eslint-disable-next-line
+  const reversedColors = legendItems.map(item => item.color).reverse();
```

Comment:
Could we either:
* add the rule here so we only disable deliberately and leave a comment here explaining why it's disabled here (the rule is fp/no-mutating-methods)
* update to `toReversed` instead of which `reverse` which should pass the fp/no-mutating-methods

I'll admit, I find this rule (and a number of the other fp/* rules) a bit too much restrictive but I'm happy to continue with them if their use is deliberate decision

Reply:
I acknowledge this is pretty nit-picky but using this opportunity to confirm our eslint rules are both used consistently and appropriately choosen